data/LinuxGamer.json

```diff
@@ -70,5 +71,8 @@
       "color": "yellow",
       "description": "I started learning JavaScript by working on some JavaScript projects, like Linkfree and Collaborafolio"
     }
+  ],
+  "testimonials": [
+    
```

Comment:
I think it's best if you remove the field currently, as it's empty, IMO.

Reply:
Done :)